crates/sui-graphql-rpc/src/types/move_package.rs

```diff
@@ -98,11 +103,19 @@ impl MovePackage {
         };
 
         let mut connection = Connection::new(false, false);
-        for (name, module) in module_range.skip(skip as usize).take(take as usize) {
+        for (name, parsed) in module_range.skip(skip as usize).take(take as usize) {
+            let Some(native) = self.native.serialized_module_map().get(name) else {
+                return Err(Error::Internal(format!(
+                    "Module '{name}' in PackageCache but not in serialized map.",
```

Comment:
I think this is missing a verb here:
```suggestion
                    "Module '{name}' exists in PackageCache but not in serialized map."
```